common-graphql/src/main/scala/queries/common/SequenceStepsGQL.scala

```diff
@@ -188,7 +185,7 @@ object SequenceStepsGQL {
       val wavelength: math.Wavelength
     }
     trait SeqFpu[Site <: SeqSite]              {
-      val builtin: Site#Fpu
+      val builtin: Option[Site#Fpu]
```

Comment:
When is this none? before setting the config?

Reply:
IDK, it changed to optional in the latest version of the API. There's now a `customMask` besides `builtin`, so I'm assuming it's one or the other. In any case, we should contemplate the `customMask` case, which I was planning to do in a future PR (it's not clear to me how to show it in the table, it has a `fileName` and `slitWidth`).

@swalker2m  ?

Reply:
Yes, when specified at all the FPU is either builtin or custom mask.  I believe the filename is actually [standardized and based on the program id](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/util/MdfConfigRule.scala#L26) (at least in today's program format).  We may need to make corresponding restrictions in the API in which case it might be sufficient to indicate "custom" and a sequence number.